addons/website_event_sale/controllers/main.py

```diff
@@ -91,3 +92,43 @@ def _prepare_event_values(self, name, event_start, event_end, address_values=Non
                 }]]
             })
         return values
+
+    @route(['/website_event_sale/ticket-price-info/<int:event_ticket_id>'],
+           type='json', auth="public", website=True, sitemap=False)
+    def get_ticket_price_info(self, event_ticket_id, **kwargs):
+        """
+        :param: int event_ticket_id: id of an event ticket
+        :param: str kwargs: optional int quantity (default 1)
+        :return: a dict containing detailed unit price of the ticket for the given quantity
+        """
+        pricelist = request.website.pricelist_id
+        event_ticket = request.env['event.event.ticket'].browse(event_ticket_id)
+        company_id = event_ticket.event_id.sudo().company_id
+        ticket_currency = company_id.currency_id
+        ctx = {}
+        if pricelist:
+            ctx.update(pricelist=pricelist.id)
+            currency_id = pricelist.currency_id
+            discount_policy = pricelist.discount_policy
+        else:
+            currency_id = ticket_currency
+            discount_policy = None
+        quantity = kwargs.get('quantity')
+        if quantity:
+            ctx['quantity'] = quantity
+
+        event_ticket = event_ticket.with_context(ctx)
+
+        return {
+            'price': ticket_currency._convert(event_ticket.price, currency_id, company_id, datetime.date.today()),
+            'price_reduce': event_ticket.price_reduce,
+            'price_reduce_taxinc': event_ticket.price_reduce_taxinc,
```

Comment:
Is it possible not to return the price values the user shouldn't see? perhaps using a `prices` key like for `currency` below?

Reply:
Yes, good idea. I have reworked that code and added some test.